iOS/SampleApp/SampleApp/Services/TextViewsInitializer.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+extension ViewController : UITextViewDelegate {
+    fileprivate static let TERMINATING_CHAR = "\n"
+    fileprivate static let DESCRIPTION_ID = "Description"
+    fileprivate static let DESCRIPTION_VALUE = "FVM description"
+    fileprivate static let DAMAGED_PARTS_ID = "DamagedCarParts"
+    fileprivate static let DAMAGED_PARTS_VALUE = "Hood; Roof;"
+    fileprivate static let DEFAULT_RESULT =
```

Comment:
I suppose it should contain empty **"mainText..."** and empty array of damaged parts.

Reply:
Yes and no, I mean, we'd return outcome you mentioned after triggering FVM and this value is used only before we do that,. It's rather used to show to the user: "Hey, you'll have your result here". Therefore, I'd not change this.